src/eval/llm_evaluator/prompts/entity_extraction_prompt.md

```
@@ -1,18 +1,32 @@
+   - consequence_variable: the variable that changes because of the trigger.
+   Example: "High credit utilization significantly increases default risk"
+   → trigger_variable: "credit utilization", consequence_variable: "default risk".
+   Example: "How do interest rates affect borrowing costs?"
+   → trigger_variable: "interest rates", consequence_variable: "borrowing costs".
 
-Input sentences:
+3. If only one term is found, leave the other empty (e.g., trigger_variable: "inflation", consequence_variable: "").
+4. Include both exact matches and conceptually related terms.
+5. Output format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)